product_docs/docs/pgd/5/quickstart/further_explore_failover.mdx

```diff
@@ -0,0 +1,410 @@
+
+```console
+bdrdb=# \d ping                                          
+                              Table "public.ping"        
+  Column   |  Type   | Collation | Nullable | Default                                                 
+-----------+---------+-----------+----------+----------------------------------                                   
+ id        | integer |           | not null | nextval('ping_id_seq'::regclass)                                    
+ node      | text    |           |          |
+ timestamp | text    |           |          |
+Indexes:
+    "ping_pkey" PRIMARY KEY, btree (id)
+```
+                                                         
+If you want to be sure that this table is replicated, you can connect to another node in the cluster and look. The `\c` command in psql lets you connect to another server. To connect to the kaftan node, run:
+
+```shell
+\c - - kaftan
+```
+
+You'll see a login message similar to this:
+
+```console
+psql.bin (15.2.0 (Debian 15.2.0-2.buster), server 15.2.0 (Debian 15.2.0-2.buster)) SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, compression: off) 
+You are now connected to database "bdrdb" as user "enterprisedb" on host "kaftan" (address "10.33.25.233") at port "5444".
+bdrdb=#
+```
+
+Run `\dt` and `\d ping`, and you'll see the same results on the kaftan node.
+
+To reconnect to the kaboom node, run:
+
+```shell
+\c - - kaboom
+```
+
+## Setting up a monitor
+
+Next, you want to monitor the activity of the ping table. Enter this SQL to display the 10 most recent entries:
+
+```sql
+select * from ping order by timestamp desc limit 10;
+```
+
+To run this command more than once, use the `\watch` command in the shell, which executes the last query at regular intervals. To update every second, enter: 
+
+```shell
+\watch 1
+```
+
+So far, there's nothing to see. You'll add activity soon.
+
+## Creating pings
+
+Return to the Barman host kapok by using **Control-b &darr; Control-b &rarr;** or **Control-b q 3**.
+
+This session is still logged into the psql session. Since you next want to run a shell script, you need to exit psql. Press **Control-d**.
+
+The shell prompt now reads:
+
+`barman@kapok:~$`
+
+If it says `admin@kapok` or `root@kapok`, run `sudo -iu barman` to become the Barman user again.
+
+The application you'll create is simple. It gets the node to write to and a timestamp for the ping. Then, as quickly as it can, it writes a new ping to the ping table.
+
+In the shell, enter:
+
+```shell
+while true; do psql -h kaftan,kaolin,kaboom -p 6432 bdrdb -c "INSERT INTO ping(node, timestamp) select node_name, current_timestamp from bdr.local_node_summary;"; done
+```
+
+In a more readable form, that is:
+
+```shell
+while true; 
+    do psql -h kaftan,kaolin,kaboom -p 6432 bdrdb -c \
+        "INSERT INTO ping(node, timestamp) select node_name, current_timestamp from bdr.local_node_summary;"
+    done
+```
```

Comment:
Why would you put it in the less-readable form?

Reply:
So it copies and pastes into the console correctly.